nbs/60_medical.imaging.ipynb

```diff
@@ -80,6 +80,56 @@
     "_all_ = ['DcmDataset', 'DcmTag', 'DcmMultiValue', 'dcmread', 'get_dicom_files']"
```

Comment:
<p>Combine these into one line, and add a <code>;</code>  at the end to avoid that extra stuff printing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>done, this looks much cleaner</p>